### DISCUSSION TITLE
spicy-snow work with TerraSARX data

### DISCUSSION THREAD

**Discussion by cirrusasf** [SnowEx/spicy-snow#73] [Ideas]
Hi, My name is Jiang Zhu, I  work for Alaska Satellite Facility. I would like to use spicy-snow  for my project. I know this is for sentinel-1c RTC data. I wonder if the package work with TerraSARX data. Do you have one or have plan to develop the workflow. If it is possible, I would like to develop the workflow.

**Comment by ZachHoppinen** [comment]
Hey Jiang, sorry just catching up on the spicy threads. Lets chat when I am up in AK in a few weeks?
